velox/common/file/benchmark/ReadBenchmark.cpp

```diff
@@ -53,16 +72,320 @@ DEFINE_validator(path, &notEmpty);
 
 namespace facebook::velox {
 
+class ReadBenchmark::Impl {
+ private:
+  enum class Mode { Pread = 0, Preadv = 1, MultiplePread = 2 };
```

Comment:
does this method translate to calling pread, preadv and multiple pread method calls in the S3 connector? 
Some description, of what this enum is intending to do will help.

Reply:
That is correct. I will add some comments.